invenio_app_rdm/config.py

```diff
@@ -481,6 +481,10 @@ def files_rest_permission_factory(obj, action):
         "task": "invenio_rdm_records.requests.access.tasks.clean_expired_request_access_tokens",
         "schedule": crontab(minute=4, hour=0),
     },
+    "delete-job-logs": {
+        "task": "invenio_jobs.logging.tasks.delete_logs",
+        "schedule": crontab(minute=5, hour=0),
```

Comment:
seems a bit too frequent :)

Reply:
This runs every day at night, deleting logs older than what specified in the config var `JOBS_LOGGING_RETENTION_DAYS` (90 days by default). Why do you think is too frequent?

Reply:
ouf my mistake, I looked only on the minutes config